back/appsync/functions/utils/models.py

```diff
@@ -0,0 +1,129 @@
+import os
```

Comment:
why its all in appsync while we  have other mail like daily mail in mail processing?